msteams-platform/sbs-gs-spfx.yml

```diff
@@ -293,7 +293,7 @@ items:
         teamsfx account login m365
         ```
 
-        Your default web browser opens to let you sign in to the account. Sign in to your account using your credentials. Close the browser when you're prompted.
```

Comment:
The first instance with full stop reads better to me. Check once with other writers.

Reply:
Hi Ashish, reverted to original instance.